### PR TITLE
cleanup remaining imports to removed winsock

### PR DESCRIPTION
**Pull request by wilzbach**
Hey guys,

_Disclaimer_: I don't know much about druntime & windows.

However I saw a couple of missing imports showing up to "winsock.d", which I couldn't find in the sources.
I couldn't find much about the history, but I assume that winsock2.d replaced it?
Don't hesitate to close, if I am wrong ;-)

### CODE REVIEW

File: src/core/sys/windows/mswsock.d [hunk 1]
```diff
@@ -13,18 +13,7 @@ version (Windows):
...
- "Version Win32_Winsock2 is deprecated; Winsock2 is now imported by default");
-
-// Pull in Winsock1 if the user has put "Win32_Winsock1" on the compile
-// line; otherwise, default to Winsock2.
-version (Win32_Winsock1) {
-    import core.sys.windows.winsock;
-} else {
-    import core.sys.windows.winsock2;
-}
+import core.sys.windows.winsock;
```
Comment (DmitryOlshansky) [inline src/core/sys/windows/mswsock.d]: Should be winsock2

Comment (wilzbach) [inline src/core/sys/windows/mswsock.d]: sorry about the type - fixed.

File: src/core/sys/windows/windows.d [hunk 2]
```diff
@@ -44,15 +44,8 @@ public import core.sys.windows.ole2;
...
-// Select correct version of winsock.  Importing the incorrect
-// module will cause a static assert to prevent problems later on.
-version (Win32_Winsock1) {
-    public import core.sys.windows.winsock;
-} else {
-    public import core.sys.windows.winsock2;
-    //public import core.sys.windows.ws2tcpip;
-}
+public import core.sys.windows.winsock2;
+//public import core.sys.windows.ws2tcpip;
```
Comment (DmitryOlshansky) [inline src/core/sys/windows/windows.d]: Kill the commented out code?

**Comment by wilzbach** [comment]
> Kill the commented out code?

wasn't sure about that. done!

**Comment by DmitryOlshansky** [comment]
Auto-merge toggled on

**Pull request merged**